lib/browser/jasmine-patch.js

```diff
@@ -0,0 +1,3 @@
+var jasminePatch = require('../jasmine/patch');
+
+jasminePatch.apply();
```

Comment:
why do we need this file? it doesn't look browser specific at all.

why not call apply from within the jasmine-patch?